### PR TITLE
license year updated

### PR DESCRIPTION
**Pull request by simonjenga**
license year updated

### CODE REVIEW

**Comment by davvd** [comment]
@simonjenga Thanks for your pull request, let me find someone who can review it

**Comment by davvd** [comment]
@carlosmiranda please review, thanks

File: qulice-ant/pom.xml [hunk 1]
```diff
@@ -99,15 +99,19 @@
             <artifactId>jcabi-aspects</artifactId>
             <scope>provided</scope>
         </dependency>
-	<dependency>
+	    <dependency>
```
Comment (carlosmiranda) [inline qulice-ant/pom.xml]: @simonjenga Why is this indented? Please revert this change.

File: qulice-ant/pom.xml [hunk 2]
```diff
@@ -99,15 +99,19 @@
...
-	    <version>1.9.4</version>
-	</dependency>
+	        <artifactId>ant</artifactId>
+	        <version>1.9.4</version>
+	    </dependency>
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
```
Comment (carlosmiranda) [inline qulice-ant/pom.xml]: @simonjenga What is this dependency for?

File: qulice-ant/src/main/java/com/qulice/ant/QuliceTask.java [hunk 3]
```diff
@@ -119,15 +124,9 @@ public void execute() {
...
-        if (this.sources == null) {
-            throw new BuildException("srcdir not defined for QuliceTask");
-        }
-        if (this.classes == null) {
-            throw new BuildException("classesdir not defined for QuliceTask");
-        }
-        if (this.classpath == null) {
-            throw new BuildException("classpath not defined for QuliceTask");
-        }
+        this.assertNotNull(this.sources, "sources");
```
Comment (carlosmiranda) [inline qulice-ant/src/main/java/com/qulice/ant/QuliceTask.java]: @simonjenga What is this for? Submit a separate pull request for this, don't include it here.

File: qulice-checkstyle/src/main/java/com/qulice/checkstyle/CheckstyleListener.java [hunk 4]
```diff
@@ -58,7 +58,7 @@
         new LinkedList<AuditEvent>();
 
     /**
-     * Public ctor.
+     * Package-private constructor.
```
Comment (carlosmiranda) [inline qulice-checkstyle/src/main/java/com/qulice/checkstyle/CheckstyleListener.java]: @simonjenga Don't include this change here. Submit a separate PR for it.

File: qulice-checkstyle/src/main/java/com/qulice/checkstyle/MethodsOrderCheck.java [hunk 5]
```diff
@@ -204,7 +204,7 @@ private void checkOrder(final Iterable<DetailAST> methods) {
         private final int order;
 
         /**
-         * Constructor.
+         * Package-private constructor.
```
Comment (carlosmiranda) [inline qulice-checkstyle/src/main/java/com/qulice/checkstyle/MethodsOrderCheck.java]: @simonjenga This too, let's do a separate PR for the javadoc changes.

File: qulice-maven-plugin/src/it/cobertura-violations/invoker.properties [hunk 6]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/cobertura-violations/invoker.properties]: @simonjenga No need to do this for the property files.

File: qulice-maven-plugin/src/it/checkstyle-violations/invoker.properties [hunk 7]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/checkstyle-violations/invoker.properties]: @simonjenga No need for property files.

File: qulice-maven-plugin/src/it/checkstyle-newlines/invoker.properties [hunk 8]
```diff
@@ -1,3 +1,35 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/checkstyle-newlines/invoker.properties]: @simonjenga No need for property files.

File: qulice-maven-plugin/src/it/checkstyle-exceptions/invoker.properties [hunk 9]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/checkstyle-exceptions/invoker.properties]: @simonjenga No need to add this header in property files.

File: qulice-gradle-plugin/src/main/resources/META-INF/gradle-plugins/qulice-plugin.properties [hunk 10]
```diff
@@ -1 +1,33 @@
+#
```
Comment (carlosmiranda) [inline qulice-gradle-plugin/src/main/resources/META-INF/gradle-plugins/qulice-plugin.properties]: @simonjenga It's not necessary for property files.

File: qulice-ant/src/it/findbugs-violations/invoker.properties [hunk 11]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-ant/src/it/findbugs-violations/invoker.properties]: @simonjenga No need to do this for property files.

File: qulice-maven-plugin/src/it/codenarc-violations/invoker.properties [hunk 12]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/codenarc-violations/invoker.properties]: @simonjenga There is no need to add it in property files.

File: qulice-maven-plugin/src/it/dependency-violations/LICENSE.txt [hunk 13]
```diff
@@ -1,2 +1,27 @@
-Some text.
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/dependency-violations/LICENSE.txt]: @simonjenga This file is deliberately designed to test whether the violation check is raised, don't change it!!!

File: qulice-maven-plugin/src/it/dependency-violations/invoker.properties [hunk 14]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/dependency-violations/invoker.properties]: @simonjenga No need to do this for property files.

File: qulice-maven-plugin/src/it/duplicate-finder-ignore-deps/invoker.properties [hunk 15]
```diff
@@ -1 +1,33 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/duplicate-finder-ignore-deps/invoker.properties]: @simonjenga No need for property files.

File: qulice-maven-plugin/src/it/duplicate-finder-violations/invoker.properties [hunk 16]
```diff
@@ -1,2 +1,34 @@
+#
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/duplicate-finder-violations/invoker.properties]: @simonjenga Same, no need for property files.

File: qulice-maven-plugin/src/it/findbugs-exclude/LICENSE.txt [hunk 17]
```diff
@@ -1 +1,28 @@
-This project has a license.
```
Comment (carlosmiranda) [inline qulice-maven-plugin/src/it/findbugs-exclude/LICENSE.txt]: @simonjenga Don't change `LICENSE.txt` within the `it/` directories!

**Comment by carlosmiranda** [comment]
@simonjenga This PR is so big, and my comments are mostly the same, so in summary:

1. Remove all other code changes not related to the `LICENSE` and submit separate PRs for them. We keep things separate on this project.
2. No need to add license headers in `.properties` files.
3. Do **not** change the license for files within `it/` folders, they were put in that way to test some specific quality checks.

Let me know when you've pushed a new commit.

**Comment by simonjenga** [comment]
@carlosmiranda Unnecessary code changes have been removed in this new pushed commit.
Other changes will be done as needed in new PRs that will be submitted later.

**Comment by carlosmiranda** [comment]
@simonjenga Thanks!

**Comment by carlosmiranda** [comment]
@rultor Looks good to merge.

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by carlosmiranda** [comment]
@yegor256 I haven't seen this error before, could you help us?

**Comment by yegor256** [comment]
@carlosmiranda it's a problem with one of the puzzles in the branch. It is probably formatted like `@todo #337.` (with a dot)

**Comment by carlosmiranda** [comment]
@yegor256 It's currently on master - can you fix it? https://github.com/teamed/qulice/blob/e69483e58bcb24e74a69911008d85577328bf5f1/qulice-ant/src/main/java/com/qulice/ant/AntEnvironment.java

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by davvd** [comment]
@carlosmiranda I just added **21 mins** to your account, many thanks for your contribution (60337461).. 810 hours and 9 mins spent here. extra minutes for review comments (c=6). +21 to your rating, your total score is [+3338](http://www.netbout.com/b/35023?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls
